lib/omnibus/packagers/bff.rb

```diff
@@ -200,6 +203,24 @@ def create_bff_file_name
       "#{safe_base_package_name}-#{project.build_version}-#{project.build_iteration}.#{safe_architecture}.bff"
     end
 
+    #
+    # Remove files with colons in their name.
+    #
+    # This is to satisfy the post-install file check done by the "sysck" utility
+    # called by installp. Without this, expect messages similar to the below
+    # (in reference to filename "/opt/harmony/embedded/share/man/man3/Tie::Memoize.3":
+    # sysck: 3001-022 The file .../share/man/man3/Tie was not found."
+    #
+    # @return [void]
+    #
+    def remove_colon_filenames
```

Comment:
This seems to be removing files that have colons in their name. The method name suggests this is removing colons from filenames though. I would suggest renaming this to `remove_files_with_colons`